README.md

```diff
@@ -27,55 +27,48 @@ If you use any [functions](https://github.com/krzkaczor/ts-essentials/blob/maste
 
 `ts-essentials` is a set of high-quality, useful TypeScript types that make writing type-safe code easier.
 
-- [Install](#Install)
-- [What's inside?](#Whats-inside)
-  - [Basic](#Basic)
-  - [Dictionaries](#Dictionaries)
-  - [Deep\* wrapper types](#Deep-wrapper-types)
-    - DeepPartial
-    - DeepRequired
-    - DeepReadonly
-    - DeepNonNullable
-    - DeepNullable
-    - DeepUndefinable
-  - [Writable & DeepWritable](#Writable)
-  - [Buildable](#Buildable)
-  - [Omit](#Omit)
-  - [StrictOmit](#StrictOmit)
-    - [Comparison between `Omit` and `StrictOmit`](#Comparison-between-Omit-and-StrictOmit)
-  - [StrictExtract](#StrictExtract)
-    - [Comparison between `Extract` and `StrictExtract`](#Comparison-between-Extract-and-StrictExtract)
-  - [DeepOmit](#DeepOmit)
-  - [DeepPick](#DeepPick)
-  - [OmitProperties](#OmitProperties)
-  - [PickProperties](#PickProperties)
-  - [NonNever](#NonNever)
-  - [Merge](#Merge)
-  - [MergeN](#MergeN)
-  - [MarkRequired](#MarkRequired)
-  - [MarkOptional](#MarkOptional)
-  - [ReadonlyKeys](#ReadonlyKeys)
-  - [WritableKeys](#WritableKeys)
-  - [OptionalKeys](#OptionalKeys)
-  - [RequiredKeys](#RequiredKeys)
+- [Install](#install)
```

Comment:
Can you please revert these changes or push them separately?

That's out of the scope of the changes

Reply:
reverted. it seems like VS Code Markdown formatter (that comes Markdown All In One extension) doesn't like plain text items like - DeepPartial  inside the table of contents and auto-deletes them. Didn't notice it at first. I disabled the extension, but maybe you should consider to modify those (make them also a links perhaps)